_data/abdoulsn/abdoulsn.md

```diff
@@ -0,0 +1,7 @@
+---
+  name: abdoulaye 
```

Comment:
Can you please remove the extra 2 space indentation from each line?

Reply:
done